src/nccl_ofi_net.c

```diff
@@ -87,8 +87,6 @@ static ncclResult_t allocate_ofi_fl(free_list_t **nccl_ofi_req_fl, size_t fl_siz
 
 	*nccl_ofi_req_fl = fl;
 
-	goto exit;
```

Comment:
We require this goto to ensure that we do not free `fl and its elements` on no error situation.

Reply:
Oh sorry, I was not paying attention to this fact
All places have been fixed in the new version of the PR.